tests/parsers/windefenderhistory.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for the Windows Defender History parser."""
+
+import unittest
+
+from plaso.parsers import windefenderhistory
+
+from tests.parsers import test_lib
+
+
+class WinDefenderHistoryUnitTest(test_lib.ParserTestCase):
+  """Tests for the Windows Defender History parser."""
+
+  def testWebFileDetection(self):
+    """Tests parsing a webfile Detection History file."""
+    parser = windefenderhistory.WinDefenderHistoryParser()
+    storage_writer = self._ParseFile([
+      'defender/'
+      'f34a0f70391bc24ecd465f713c6b068cc24dd05d71994924abbb438ba9eecf62'
+    ], parser)
+
+    number_of_warnings = storage_writer.GetNumberOfAttributeContainers(
+        'extraction_warning')
+    self.assertEqual(number_of_warnings, 0)
+
+    number_of_warnings = storage_writer.GetNumberOfAttributeContainers(
+        'recovery_warning')
+    self.assertEqual(number_of_warnings, 0)
+
+    for event in storage_writer.GetEvents():
+      event_data = self._GetEventDataOfEvent(storage_writer, event)
+      self.assertEqual(
+          'C:\\Users\\Luqman\\Downloads\\Ransomware.Jigsaw.zip',
+          event_data.filename)
+      self.assertEqual(
+          'C:\\Users\\Luqman\\Downloads\\Ransomware.Jigsaw.zip|http://10.1.20.13:8000/Ransomware.Jigsaw.zip|pid:1020,ProcessStart:132525829301711007', # pylint: disable=line-too-long
```

Comment:
don't use `# pylint: disable=line-too-long`

split the string using bounding parenthesis

Reply:
Done